pkg/wait/conditions.go

```diff
@@ -280,9 +280,19 @@ func AreAllPodsInSuccessfulPhase(ctx context.Context, kubeClient *client.Client)
 
 // AreNoPodsCrashLooping checks that all pods within the cluster have fewer than the provided number of restarts
 func AreNoPodsCrashLooping(ctx context.Context, kubeClient *client.Client, maxRestartCount int32) WaitCondition {
+	return AreNoPodsCrashLoopingExcluding(ctx, kubeClient, maxRestartCount, []string{})
+}
+
+// AreNoPodsCrashLoopingExcluding checks that all pods within the cluster have fewer than the provided number of restarts
+func AreNoPodsCrashLoopingExcluding(ctx context.Context, kubeClient *client.Client, maxRestartCount int32, excludePodLabels []string) WaitCondition {
 	return func() (bool, error) {
 		podList := &corev1.PodList{}
-		err := kubeClient.List(ctx, podList)
+		podListOptions := []cr.ListOption{}
+		for _, exclude := range excludePodLabels {
```

Comment:
Is the key enough, or do we need label key+value? I expected passing something like `["app", "cluster-autoscaler"]`.

Reply:
ah damn! I misunderstood the code and thought this could be used to exclude key/value labels but looks like it's not. Let me fix that. Nice catch!